spec/docker-compose.yml

```diff
@@ -56,5 +56,5 @@ services:
       args:
         BOLT_PASSWORD: bolt
     ports:
-      - "24985:5985"
-      - "24986:5986"
+      - "45985:5985"
```

Comment:
Purely out of curiosity is there a reason for changing these ports?

Reply:
Noted in commit message at https://github.com/puppetlabs/bolt/pull/1087/commits/c4c8a2783fd7831de9a0edfe3a6339816f8b5449 ... its to keep synced up around 5985 / 5986, which are the winrm ports you'd typically be grepping for.